renovate.json

```diff
@@ -12,9 +12,6 @@
   "schedule": [
```

Comment:
Remove `dependencyDashboard` from `extends` -- already included in the base config.